src/Framework/TestCase.php

```diff
@@ -1104,8 +1116,20 @@ final protected function runTest(): mixed
     {
         $testArguments = array_merge($this->data, array_values($this->dependencyInput));
 
+        $capture          = tmpfile();
+        $errorLogPrevious = ini_set('error_log', stream_get_meta_data($capture)['uri']);
```

Comment:
I saw your php-src feature request to support in memory stream. I think stream wrapper would work here - https://www.php.net/manual/en/stream.streamwrapper.example-1.php

Reply:
Also creating tmp. file for every testcase is far from ideal.

Reply:
feel free to improve the implementation with a followup PR

Reply:
> I think stream wrapper would work here - https://www.php.net/manual/en/stream.streamwrapper.example-1.php

sadly even this is currently not possible - https://github.com/php/php-src/issues/18530